### PR TITLE
chore(ADOPTERS.md): addition of Civo

### PR DESCRIPTION
**Pull request by AlexsJones**
Signed-off-by: Alex <alexsimonjones@gmail.com>

Addition of [Civo](https://www.civo.com) into Adopters list.
Thanks

```release-note
NONE
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ This is a likely incomplete list of KubeVirt adopters - end-users and distributo
 | H3C | 2019 | We distribute KubeVirt as part of CloudOS to enable VM workloads on Kubernetes at customer sites . Follow the  [link](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/) for more product information |
 | NVIDIA | 2018 | NVIDIA's latest computing platform is built on open-source projects like Kubernetes and KubeVirt to power products like [GeForce NOW](https://www.nvidia.com/en-us/geforce-now/) with more to come.
 | [CoreWeave](https://www.coreweave.com) | 2020 | A Kubernetes native cloud provider with focus on GPUs at scale. KubeVirt allows us to co-locate non-containerizable workloads such as Virtual Desktops next to compute intensive containers executing on bare metal. All orchestrated via the Kubernetes API leveraging the same network policies and persistent volumes for both VM and containerized workloads. |
+| [Civo](https://www.civo.com) | 2020 | We are using Kubevirt as part of our stack to enable tenant cluster provisioning within Civo cloud. |
```
Comment (dankenigsberg) [inline ADOPTERS.md:12]:
```suggestion
| [Civo](https://www.civo.com) | 2020 | We are using KubeVirt as part of our stack to enable tenant cluster provisioning within Civo cloud. |
```

**Review by dankenigsberg — CHANGES_REQUESTED** [review]
/ok-to-test

**Comment by AlexsJones** [comment]
> @AlexsJones: The following tests **failed**, say `/retest` to rerun all failed tests:
> 
> Test name	Commit	Details	Rerun command
> pull-kubevirt-e2e-k8s-1.17-rook-ceph	[dd3d9a7](https://github.com/kubevirt/kubevirt/commit/dd3d9a7c46a207e65a491496120841030f6e092c)	[link](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5516/pull-kubevirt-e2e-k8s-1.17-rook-ceph/1386280502528315392)	`/test pull-kubevirt-e2e-k8s-1.17-rook-ceph`
> pull-kubevirt-e2e-k8s-1.18	[dd3d9a7](https://github.com/kubevirt/kubevirt/commit/dd3d9a7c46a207e65a491496120841030f6e092c)	[link](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5516/pull-kubevirt-e2e-k8s-1.18/1386280502436040704)	`/test pull-kubevirt-e2e-k8s-1.18`
> Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md). If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).

/retest

**Review by davidvossel — COMMENTED** [review]
/approve

File: ADOPTERS.md (hunk 1 above)
Comment (dankenigsberg) [reply ADOPTERS.md:12]: @AlexsJones would you accept this suggestion?

**Pull request merged**